app/models/labware.rb

```diff
@@ -2,6 +2,7 @@
 
 # Labware represents a physical object which moves around the lab.
 # It has one or more receptacles.
+# rubocop:disable Metrics/ClassLength
```

Comment:
I think differently about this. Having the rule makes you think, check what can be shorter,  whether you should normalise, whether you are over-engineering, or whether you should separate some code into another module. I would like different length configurations for different categories of classes though.

Reply:
I agree.  Where it can be left enabled, that is my top preference.  The problem I see here is that we're adding an exception in this file that, down the line, will get ignored and forgotten so the value of the rule has been muted.